maintenance/aws-janitor/resources/dhcp_options.go

```diff
@@ -131,6 +131,10 @@ func defaultLookingDHCPOptions(dhcp *ec2.DhcpOptions, region string) bool {
 		switch *conf.Key {
 		case "domain-name":
 			var domain string
+			// TODO(akutz): Should this be updated to regions.Default, or is
+			// this relying on the default region for EC2 for North America?
+			// Because EC2's default region changed from us-east-1 to us-east-2
+			// depending on when the account was created.
```

Comment:
I believe the comparison against "us-east-1" here is correct, since this is attempting to work around the way that ec2 computes the internal DNS names for hosts created outside of us-east-1 vs inside of us-east-1.

Reply:
Hi @detiber,

That was my thought as well. I searched the `aws-janitor` directory recursively for `us-east-1` and replaced everything with this PR's changes. However, this instance of the string looked like it was what you're describing, and I wanted to be sure before I changed it.